doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -315,6 +315,10 @@ img.sk-who-uses-carousel-img {
   max-width: 50%;
```

Comment:
What do you think of adding a little space between the image and the testimonial, so there is less whitespace between the testimonial and the "More testimonials" link (and more whitespace between the logo and the testimonial):

```css
img.sk-who-uses-carousel-img {
  max-height: 100px;
  max-width: 50%;
  margin-bottom: 0.75rem;
}
```

Reply:
How about:

```css
img.sk-who-uses-carousel-img {
   height: 80px;
   margin-bottom: 0.5rem;
}
```
and remove the `min-height` from `#carouselExampleSlidesOnly`?

Reply:
Just checked... the link is still moving up and down (as expected... ;)) because the height of the caption is not known in advance ....
C'mon ...You already approved it ;)